pkg/shell/superuser.jsx

```diff
@@ -303,7 +303,7 @@ const SuperuserDialogs = ({ superuser_proxy, host, create_trigger }) => {
         return;
 
     const show = (superuser_proxy.Current != "root" && superuser_proxy.Current != "init" &&
-                  superuser_proxy.Bridges.length > 0);
+                  (superuser_proxy.Bridges?.length ?? 0) > 0);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19486-20231016-045955-b3ccd78d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)